dotnetv3/README.md

```diff
@@ -129,21 +71,45 @@ If you want more information, run:
 dotnet test -l "console;verbosity=detailed"
 ```
 
+To specify either unit or integration tests only, use the following category filters with the desired verbosity:
+```
+dotnet test --filter Category=Unit -l "console;verbosity=detailed"
+```
+or
+```
+dotnet test --filter Category=Integration -l "console;verbosity=detailed"
+```
+
 ## Docker image (Beta)
 This example code will soon be available in a container image
 hosted on [Amazon Elastic Container Registry (ECR)](https://docs.aws.amazon.com/AmazonECR/latest/userguide/what-is-ecr.html). This image will be pre-loaded 
-with all .NET examples with dependencies pre-resolved, allowing you to explore
+with all .NET examples ready to build and run, allowing you to explore
```

Comment:
Should this still say "will soon" and "will be"? Or is it available now, just in beta form?

Reply:
The updated version is still "will soon be available", we are holding off on those ECR updates for now. What is there now is only compatible with .NET 6, so I believe this wording is still accurate.